### ISSUE TITLE
Query Builder creates invalid distinct query after sorting

### ISSUE THREAD

**Issue by senior** [metabase/metabase#7042]
I was able to reproduce this on stats with 0.28.2. Issue a "Raw Data" query, then click on a column and sort descending. Now click on the same column and click "Distincts". This will cause an error due to an invalid SQL query
```
org.postgresql.util.PSQLException: ERROR: column "metabase_field.visibility_type" must appear in the GROUP BY clause or be used in an aggregate function Position: 236
```

**Comment by flamber** [comment]
Hi @senior 
I cannot reproduce this issue on 0.31.2 with Postgres 11.2
![screencast 2019-02-18 18 57 42](https://user-images.githubusercontent.com/1447303/52983192-8d094880-33af-11e9-86b7-89a776bde411.gif)

**Comment by mazameli** [comment]
Closing this as fixed as I can't reproduce it in 0.33.0 either. Must have gotten fixed along the way. @tlrobinson or @kdoh, do you know if there's a corresponding test for this yet?
